string NAME=hovered

ObjectDrone d

bool truth_value
